.github/workflows/release.yml

```diff
@@ -79,21 +75,21 @@ jobs:
         if [ "${{ matrix.os }}" = "windows-latest" ]; then
           crx2rnx="target/${{ matrix.target }}/release/crx2rnx.exe"
           rnx2crx="target/${{ matrix.target }}/release/rnx2crx.exe"
-          rinex-cli="target/${{ matrix.target }}/release/rinex-cli.exe"
+          rinexcli="target/${{ matrix.target }}/release/rinex-cli.exe"
           rnx2cggtts="target/${{ matrix.target }}/release/rnx2cggtts.exe"
-          ublox-rnx="target/${{ matrix.target }}/release/ublox-rnx.exe"
+          ubloxrnx="target/${{ matrix.target }}/release/ublox-rnx.exe"
         else
           crx2rnx="target/${{ matrix.target }}/release/crx2rnx"
           rnx2crx="target/${{ matrix.target }}/release/rnx2crx"
-          rinex-cli="target/${{ matrix.target }}/release/rinex-cli"
+          rinexcli="target/${{ matrix.target }}/release/rinex-cli"
           rnx2cggtts="target/${{ matrix.target }}/release/rnx2cggtts"
-          ublox-rnx="target/${{ matrix.target }}/release/ublox-rnx"
+          ubloxrnx="target/${{ matrix.target }}/release/ublox-rnx"
```

Comment:
Doesn't the target directly get doubled? It's both here and in the cp below.

Reply:
Hello,
not sure I understand what you mean.
We have the build step, and then I copy the generated binaries into an "archive" folder, to be zipped later on

Reply:
`$crx2rnx` includes `target/whatever/release`, it gets stored into `$CRX2RNX`, then you do `cp target/whatever/release/$CRX2RNX`.

Reply:
😆 so obvious I did not see it. Thanks ! 👍 
In this current state, the binaries will exist and will be zipped, the release gets created,
but we still need to retrieve the zipped binaries and use a job dedicated to release upload.

The way [I did it at the bottom of this script](https://github.com/gwbres/cggtts/blob/main/.github/workflows/release.yml), is to download all zipped artifacts and then trigger the upload job